src/app/pages/receive/receive-dialog.tsx

```diff
@@ -0,0 +1,130 @@
+import toast from 'react-hot-toast';
+import { useLocation, useNavigate } from 'react-router-dom';
+
+import get from 'lodash.get';
+
+import { RouteUrls } from '@shared/route-urls';
+
+import { useAnalytics } from '@app/common/hooks/analytics/use-analytics';
+import { useClipboard } from '@app/common/hooks/use-copy-to-clipboard';
+import { useLocationState } from '@app/common/hooks/use-location-state';
+import { useBackgroundLocationRedirect } from '@app/routes/hooks/use-background-location-redirect';
+import { useZeroIndexTaprootAddress } from '@app/store/accounts/blockchain/bitcoin/bitcoin.hooks';
+import { useCurrentAccountNativeSegwitAddressIndexZero } from '@app/store/accounts/blockchain/bitcoin/native-segwit-account.hooks';
+import { useCurrentAccountStxAddressState } from '@app/store/accounts/blockchain/stacks/stacks-account.hooks';
+import { Dialog } from '@app/ui/components/containers/dialog/dialog';
+import { BigTitle } from '@app/ui/components/containers/headers/header';
+import { Tabs } from '@app/ui/components/tabs/tabs';
+
+import { ReceiveCollectibles } from './components/receive-collectibles';
+import { ReceiveTokens } from './components/receive-tokens';
+
+type ReceiveDialog = 'full' | 'collectible';
+
+interface ReceiveDialogProps {
+  type?: 'full' | 'collectible';
+}
+// ideally all tabs should match but choose-fee has a different variation
+export const receiveTabStyle = {
```

Comment:
Just a pref here, but I would prefer the export go above the component props.

Reply:
Sure, makes sense, I'll update it 👍